zenoh-ts/.npmrc

```diff
@@ -0,0 +1 @@
+@ZettaScaleLabs:registry=https://npm.pkg.github.com
```

Comment:
This should be the npm for eclipse not the private github one

Reply:
This should be `@EclipseZenoh:registry=https://npm.pkg.github.com` otherwise I'm pretty sure it will not work